app.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+function Project (name, description,url){
```

Comment:
Good job adding a constructor. Remember to start using your ``projects.js`` file in your future reincarnations of this blog so that your constructor is actually being used. See today's starter code here to get an idea of how to utilize your data and instantiate it as a new Project (in the starter's code case, it is instantiated as a new Article on Line 29).